c8run/package.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"c8run/internal/archive"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+func Clean(camundaVersion string, elasticsearchVersion string) {
+	os.RemoveAll("elasticsearch-" + elasticsearchVersion)
+	os.RemoveAll("camunda-zeebe-" + camundaVersion)
+
+	_, err := os.Stat(filepath.Join("log", "camunda.log"))
+	if errors.Is(err, os.ErrNotExist) {
+		os.Remove(filepath.Join("log", "camunda.log"))
+	}
+	_, err = os.Stat(filepath.Join("log", "connectors.log"))
+	if errors.Is(err, os.ErrNotExist) {
+		os.Remove(filepath.Join("log", "connectors.log"))
+	}
+	_, err = os.Stat(filepath.Join("log", "elasticsearch.log"))
+	if errors.Is(err, os.ErrNotExist) {
+		os.Remove(filepath.Join("log", "elasticsearch.log"))
+	}
```

Comment:
3 remarks:
- style: use for loop
- correctness : seems like the files are being deleted only when they don't exist
- approach : prefer something simpler which allows to clean undesired files  using a single command. It would be nice if the files needing cleaning would be under a specific parent that cou can simply remove

Reply:
I addressed style + correctness in 3e472c772df .  I will think on "approach" later.

Reply:
I included `./c8run clean` which will delete these files.